src/ZfcRbac/Factory/IsGrantedViewHelperFactory.php

```diff
@@ -31,14 +35,25 @@
 class IsGrantedViewHelperFactory implements FactoryInterface
 {
     /**
-     * {@inheritDoc}
+     * @param ContainerInterface $container
+     * @param string $requestedName
+     * @param array|null $options
      * @return IsGranted
      */
-    public function createService(ServiceLocatorInterface $serviceLocator)
+    public function __invoke(ContainerInterface $container, $requestedName, array $options = null)
     {
         /* @var \ZfcRbac\Service\AuthorizationService $authorizationService */
-        $authorizationService = $serviceLocator->getServiceLocator()->get('ZfcRbac\Service\AuthorizationService');
+        $authorizationService = $container->get('ZfcRbac\Service\AuthorizationService');
```

Comment:
same as above. ZF3 plugin manager send the main locator not the plugin manager itself to the factory